doc_templates/phylum_extension_new.md

```diff
@@ -1,16 +1,7 @@
----
-title: phylum extension new
-category: 6255e67693d5200013b1fa3e
-parentDoc: 62d04d1ec90dcf008e476330
-hidden: false
----
+{PH-HEADER}
 
 {PH-MARKDOWN}
 
 ### Details
 
-This command will create a "Hello, World!" extension skeleton that can be
-modified for a specific use. See the [Extension Quickstart] guide for more
-detail.
-
-[Extension Quickstart]: https://docs.phylum.io/docs/extension_quickstart
+This command will create a "Hello, World!" extension skeleton that can be modified for a specific use. See the [Extension Quickstart](../extensions/extension_quickstart.md) guide for more detail.
```

Comment:
Don't want to be overly pedantic but this is technically already applying your preferred markdown style. Should probably be reverted.

Reply:
Oops...missed this one during the revert. It will be changed.